### PR TITLE
🌐 Translate to dutch

### PR DESCRIPTION
**Pull request by coyotte508**
Fix https://github.com/B2Bitcoin/beBOP/issues/1404

### CODE REVIEW

File: src/lib/translations/nl.json [hunk 1]
```diff
@@ -0,0 +1,672 @@
...
+		"list": "Lijst met uitdagingen",
+		"moneyAmount": {
+			"goalOvershot": "<0></0> op een doel van <1></1> ! \nJe bent geweldig !"
+		},
+		"numProducts": {
+			"one": "{count} product",
+			"other": "{count} producten",
+			"zero": "{count} producten"
+		},
+		"singleTitle": "Uitdaging {naam}",
```
Comment (coyotte508) [inline src/lib/translations/nl.json:57]:
```suggestion
		"singleTitle": "Uitdaging {name}",
```

File: src/lib/translations/nl.json [hunk 2]
```diff
@@ -0,0 +1,672 @@
...
+			"identity": "Identiteit",
+			"logout": "Uitloggen",
+			"orders": "Bestellingen",
+			"signInWith": "Inloggen met {provider}"
+		},
+		"password": {
+			"pwned": "Uw wachtwoord is onveilig. Het is aangetroffen bij {count} datalekken. \nStel alstublieft uw wachtwoord opnieuw in."
+		},
+		"session": {
+			"email": "E-mail: {e-mail}",
```
Comment (coyotte508) [inline src/lib/translations/nl.json:427]:
```suggestion
			"email": "E-mail: {email}",
```

File: src/lib/translations/nl.json [hunk 3]
```diff
@@ -0,0 +1,672 @@
...
+			"identityPrefill": "U kunt uw postadres en uw contactadres vooraf invullen. \nAls u een bestelling plaatst, zijn deze gegevens vooraf ingevuld. \nAls u later besluit een account aan te maken, worden deze gegevens bewaard voor toekomstige bestellingen.",
+			"npub": "Npub: {npub}",
+			"title": "Sessie",
+			"userId": "Gebruikers-ID: {user-ID}"
```
Comment (coyotte508) [inline src/lib/translations/nl.json:431]:
```suggestion
			"userId": "Gebruikers-ID: {userId}"
```

File: src/lib/translations/nl.json [hunk 4]
```diff
@@ -0,0 +1,672 @@
...
+			"proforma": "Dit is een proforma factuur. \nEr heeft nog geen betaling plaatsgevonden. \nBij betaling wordt de definitieve factuur gegenereerd.",
+			"proformaInvoiceNumber": "PROFORMA-{ordernummer}-{betalingsindex}",
+			"quantity": "Aantal",
+			"remainingAmount": "Resterend bedrag te betalen",
+			"summaryTitle": "Samenvatting van de bestelling",
+			"totalExcVat": "Totaal Excl. \nVAT",
+			"totalInclVat": "Totaal Incl. \nVAT",
+			"totalVat": "Totale BTW",
+			"unitPrice": "Eenheid prijs",
+			"vatFreeReason": "Vrijgesteld van btw: {reden}"
```
Comment (coyotte508) [inline src/lib/translations/nl.json:546]:
```suggestion
			"vatFreeReason": "Vrijgesteld van btw: {reason}"
```

File: src/lib/translations/nl.json [hunk 5]
```diff
@@ -0,0 +1,672 @@
...
+		},
+		"receiptFullyPaid": "Bon genereren (volledig betaald)",
+		"receiptPending": "Ontvangst genereren (betaling in behandeling)",
+		"related": {
+			"invoice": "✅ Factuurnr.{factuurnummer}",
```
Comment (coyotte508) [inline src/lib/translations/nl.json:551]:
```suggestion
			"invoice": "✅ Factuurnr.{invoiceNumber}",
```

File: src/lib/translations/nl.json [hunk 6]
```diff
@@ -0,0 +1,672 @@
...
+			"invoiceFailed": "❌ Factuurnr.{factuurnummer}",
```
Comment (coyotte508) [inline src/lib/translations/nl.json:552]:
```suggestion
			"invoiceFailed": "❌ Factuurnr.{invoiceNumber}",
```

**Review by Tirodem — APPROVED** [review]

**Comment by coyotte508** [comment]
hmm @Tirodem should have applied the suggestions before merging

**Comment by Tirodem** [comment]
> hmm @Tirodem should have applied the suggestions before merging

Woopsie :x

**Pull request merged**